client/me/purchases/billing-history/main.tsx

```diff
@@ -39,6 +41,12 @@ export function BillingHistoryContent( {
 }
 
 function BillingHistory(): JSX.Element {
+	const translate = useTranslate();
+	const { vatDetails } = useVatDetails();
+	const editVatText = translate( 'Edit VAT details (for EU and UK only)' );
+	const addVatText = translate( 'Add VAT details (for EU and UK only)' );
```

Comment:
Does the average user know what "EU" and "UK" are?

If spelling it out is too long, possibly something like "European businesses only" or "for Europe only" would be OK here -- I know it's not offered for every country in Europe, but that could also be explained in more detail on the next page.